content/ui-patterns/forms.mdx

```diff
@@ -0,0 +1,387 @@
+---
+title: Forms
+---
+
+import {Box, Heading, Link} from '@primer/components'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+    Forms are used to complete tasks that require data input from the user. For example: creating a new repo, configuring settings, and logging in.
+</Box>
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+    Primer's form design guidelines aim to minimize the effort and cognitive load required to complete a task that involves a form.
+</Box>
```

Comment:
@mperrotti Can we simplify this leading description and make it into a single paragraph?

> Forms are used to complete tasks that require data input from the user. For example: creating a new repo, configuring settings, and logging in.
>
> Primer's form design guidelines aim to minimize the effort and cognitive load required to complete a task that involves a form.